File: FamilyVacationPlanner/src/FamilyVacationPlanner.WebApp/projects/family-vacation-planner/src/app/models/packing-list.ts
export interface PackingList {
  packingListId: string;
  tripId: string;
  itemName: string;
  isPacked: boolean;
  createdAt: string;
}

export interface CreatePackingListCommand {
  tripId: string;
  itemName: string;
  isPacked: boolean;
}

export interface UpdatePackingListCommand {
  packingListId: string;
  tripId: string;
  itemName: string;
  isPacked: boolean;
}

File: FamilyVacationPlanner/src/PackingLists/PackingLists.Core/Models/PackingList.cs
namespace PackingLists.Core.Models;

public class PackingList
{
    public Guid PackingListId { get; set; }
    public Guid TenantId { get; set; }
    public Guid TripId { get; set; }
    public string ItemName { get; set; } = string.Empty;
    public bool IsPacked { get; set; }
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}
